content/cloud-docs/api-docs/run-tasks-integration.mdx

```diff
@@ -0,0 +1,145 @@
+
+### Sample Payload
+
+```json
+{
+  "payload_version": 1,
+  "access_token": "4QEuyyxug1f2rw.atlasv1.iDyxqhXGVZ0ykes53YdQyHyYtFOrdAWNBxcVUgWvzb64NFHjcquu8gJMEdUwoSLRu4Q",
+  "stage": "post_plan",
+  "is_speculative": false,
+  "task_result_id": "taskrs-2nH5dncYoXaMVQmJ",
+  "task_result_enforcement_level": "mandatory",
+  "task_result_callback_url": "https://app.terraform.io/api/v2/task-results/5ea8d46c-2ceb-42cd-83f2-82e54697bddd/callback",
+  "run_app_url": "https://app.terraform.io/app/hashicorp/my-workspace/runs/run-i3Df5to9ELvibKpQ",
+  "run_id": "run-i3Df5to9ELvibKpQ",
+  "run_message": "Triggered via UI",
+  "run_created_at": "2021-09-02T14:47:13.036Z",
+  "run_created_by": "username",
+  "workspace_id": "ws-ck4G5bb1Yei5szRh",
+  "workspace_name": "tfr_github_0",
+  "workspace_app_url": "https://app.terraform.io/app/hashicorp/my-workspace",
+  "organization_name": "hashicorp",
+  "plan_json_api_url": "https://app.terraform.io/api/v2/plans/plan-6AFmRJW1PFJ7qbAh/json-output",
+  "vcs_repo_url": "https://github.com/hashicorp/terraform-random",
+  "vcs_branch": "main",
+  "vcs_pull_request_url": null,
+  "vcs_commit_url": "https://github.com/hashicorp/terraform-random/commit/7d8fb2a2d601edebdb7a59ad2088a96673637d22"
+}
+```
+
+### Request Headers
+
+The POST request submits the following properties as the request headers.
+
+|          Name          |                   Value                    |                                                                                                                    Description                                                                                                                    |
+| ---------------------- | ------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `Content-Type`         | `application/json`                         | Specifies the type of data in the request body                                                                                                                                                                                                    |
+| `User-Agent`           | `TFC/1.0 (+https://app.terraform.io; TFC)` | Identifies the request is coming from Terraform Cloud                                                                                                                                                                                             |
+| `X-TFC-Task-Signature` | string                                     | If the run task is configured with a [HMAC Key](cloud-docs/integrations/run-tasks#securing-your-run-task), this header contains the signed SHA512 sum of the request payload using the configured HMAC key. Otherwise, this is an empty string. |
```

Comment:
```suggestion
| `X-TFC-Task-Signature` | string                                     | If the run task is configured with an [HMAC Key](cloud-docs/integrations/run-tasks#securing-your-run-task), this header contains the signed SHA512 sum of the request payload using the configured HMAC key. Otherwise, this is an empty string. |
```